alerts/openshift-container-storage-operator/CephMdsCpuUsageHigh.md

```diff
@@ -0,0 +1,31 @@
+# MDSCacheUsageHigh
+
+## Meaning
+
+Ceph MDS cpu usage for the MDS daemon has exceeded the thereshold for adaquete performance.
+
+## Impact
+
+MDS serves filesystem metadata. The MDS is crucial for any file creation, rename, deletion and update operations. When the load is more than the CPU resource for the MDS, then we get the alert.
+
+## Diagnosis
+
+We need to either increase the allocated CPU or run multiple active MDS . The blow command describes how to set the number of allocated CPU for MDS server.
+
+```bash
+oc patch -n openshift-storage storagecluster ocs-storagecluster \
+    --type merge \
+    --patch '{"spec": {"resources": {"mds": {"limits": {"cpu": "8"},"requests": {"cpu": "8"}}}}}'
+```
+
+In order to run multiple active MDS servers, use below command
+
+```bash
+oc patch -n openshift-storage cephfilesystem ocs-storagecluster-cephfilesystem \
+    --type merge \
+    --patch '{"spec": {"metadataServer": {"activeCount": 2}}}'
+```
```

Comment:
This should be in mitigation section.
Diagnosis should explain how to diagnose the issue. What is the trigger condition for this alert?

Reply:
Agree with Umanga. 
Move the current "diagnosis" content to  "mitigation"... and in the "diagnosis" section explain how the user can see by himself the problem, in this case the high MDS pods CPU usage.

Reply:
ack